travel-advice-publisher/docker-compose.yml

```diff
@@ -0,0 +1,65 @@
+version: '3.7'
+
+x-travel-advice-publisher: &travel-advice-publisher
+  build:
+    context: .
+    dockerfile: travel-advice-publisher/Dockerfile
+  image: travel-advice-publisher
+  volumes:
+    - ..:/govuk:delegated
+    - bundle:/usr/local/bundle
+    - home:/home/build
+  working_dir: /govuk/travel-advice-publisher
+
+services:
+  travel-advice-publisher-default:
+    <<: *travel-advice-publisher
+    privileged: true
+    depends_on:
+      - mongo
+      - redis
+    environment:
+      MONGODB_URI: "mongodb://mongo/travel-advice-publisher"
+      TEST_MONGODB_URI: "mongodb://mongo/travel-advice-publisher-test"
+      REDIS_URL: redis://redis
+
+  travel-advice-publisher-backend: &travel-advice-publisher-backend
+    <<: *travel-advice-publisher
+    depends_on:
+      - mongo
+      - redis
+      - publishing-api-backend
+      - asset-manager-backend
+      - content-store-live
+      - nginx-proxy
+    environment:
+      MONGODB_URI: "mongodb://mongo/travel-advice-publisher"
+      TEST_MONGODB_URI: "mongodb://mongo/travel-advice-publisher-test"
```

Comment:
Minor, but we shouldn't need this particular env var outside of the default stack, since we're (hopefully) not running the tests here.